cornedbeef/src/main/java/com/swiftkey/cornedbeef/CoachMark.java

```diff
@@ -144,12 +153,12 @@ public void dismiss() {
         mAnchor.destroyDrawingCache();
         mAnchor.getViewTreeObserver().removeOnPreDrawListener(mPreDrawListener);
         mPopup.dismiss();
-        
```

Comment:
can you checkout these diffs. It makes looking at this hard to see what really changed.